src/protocol/ipa/mod.rs

```diff
@@ -860,19 +864,25 @@ pub mod tests {
                 .reconstruct();
 
             let snapshot = world.metrics_snapshot();
-            let records_sent = snapshot.get_counter(RECORDS_SENT);
-            let semi_honest_baseline = if per_user_cap == 1 {
-                RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_1
+            let (records_baseline, bytes_baseline) = if per_user_cap == 1 {
+                (
+                    RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_1,
+                    FIELD_SIZE * RECORDS_SENT_SEMI_HONEST_BASELINE_CAP_1,
```

Comment:
I don’t think this is right. Currently IPA runs two separate validators, each over a subset of the protocol, so some records are sent in field 1 and others in field 2

Reply:
yep, my rebase was 4 hours old and stale already!